### PR TITLE
Update debian mirrors to fix audit

### PR DESCRIPTION
**Pull request by scpeters**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I noticed an audit failure related to debian mirrors during #38307, so I thought I would update a few other formulae. I checked and these all have clean audits with `--strict --online` without being installed. We shouldn't need to pull new bottles for this.

### CODE REVIEW

File: Formula/mmv.rb [hunk 1]
```diff
@@ -17,7 +16,7 @@ class Mmv < Formula
   end
 
   patch do
-    url "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/m/mmv/mmv_1.01b-15.diff.gz"
```
Comment (scpeters) [inline Formula/mmv.rb]: this file is no longer on either deb server

Comment (scpeters) [reply Formula/mmv.rb]: https://launchpadlibrarian.net/27360154/mmv_1.01b-15.diff.gz

Comment (scpeters) [reply Formula/mmv.rb]: perhaps we should upload this to https://github.com/Homebrew/formula-patches ?

**Comment by scpeters** [comment]
castxml fails to compile against the latest version of llvm

~~~
src/castxml.cxx:65:47: error: no member named 'GetArgumentVector' in 'llvm::sys::Process'
  if (std::error_code e = llvm::sys::Process::GetArgumentVector(
                          ~~~~~~~~~~~~~~~~~~~~^
1 error generated.
~~~

that API was changed in https://github.com/llvm-mirror/llvm/commit/3e79068e735685d0823c48cd80eda14347cac237

File: Formula/mmv.rb (hunk 1 above)
Comment (scpeters) [reply Formula/mmv.rb]: https://github.com/Homebrew/formula-patches/pull/267

**Comment by scpeters** [comment]
castxml error may be fixed by https://github.com/CastXML/CastXML/commit/a345f628237a36c9ca55364760e41ff7a936e7da

File: Formula/mmv.rb (hunk 1 above)
Comment (scpeters) [reply Formula/mmv.rb]: uncompressed patch in https://github.com/Homebrew/formula-patches/pull/269

**Review by MikeMcQuaid — APPROVED** [review]

**Pull request closed without merging**